<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>810</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>810</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>19</y>
     <width>811</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>24</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 700 24pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>PKU 课 程 学 生 信 息 管 理 系 统</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>751</width>
     <height>401</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>591</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="AddButton">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>添加信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteButton">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>删除信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="EditButton">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>修改信息</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="SaveButton">
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;等线&quot;;</string>
      </property>
      <property name="text">
       <string>保存修改</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>481</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="LessButton">
      <property name="text">
       <string>按总评成绩升序排列</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="GreaterButton">
      <property name="text">
       <string>按总评成绩降序排列</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
